src/servers/src/otlp/trace/v0.rs

```diff
@@ -131,31 +75,31 @@ pub fn write_span_to_row(writer: &mut TableData, span: TraceSpan) -> Result<()>
             ValueData::TimestampNanosecondValue(span.end_in_nanosecond as i64),
         ),
         make_column_data(
-            "duration_nano",
+            DURATION_NANO_COLUMN,
             ColumnDataType::Uint64,
             ValueData::U64Value(span.end_in_nanosecond - span.start_in_nanosecond),
         ),
     ];
     row_writer::write_fields(writer, fields.into_iter(), &mut row)?;
 
     if let Some(service_name) = span.service_name {
-        row_writer::write_tag(writer, "service_name", service_name, &mut row)?;
+        row_writer::write_tag(writer, SERVICE_NAME_COLUMN, service_name, &mut row)?;
     }
 
     // tags
     let iter = vec![
-        ("trace_id", span.trace_id),
-        ("span_id", span.span_id),
-        ("parent_span_id", span.parent_span_id),
+        (TRACE_ID_COLUMN, span.trace_id),
+        (SPAN_ID_COLUMN, span.span_id),
+        (PARENT_SPAN_ID_COLUMN, span.parent_span_id),
     ]
     .into_iter()
     .map(|(col, val)| (col.to_string(), val));
     row_writer::write_tags(writer, iter, &mut row)?;
```

Comment:
Same as below for setting tags

Reply:
This v0 is kept as it was. We will deprecated this data model so I'm not updating this